apps/ledger-live-mobile/src/dynamicContent/utils.tsx

```diff
@@ -28,16 +28,10 @@ export const filterByType = (array: BrazeContentCard[], type: ContentCardsType)
   array.filter(elem => elem.extras.type === type);
 
 export const compareCards = (a: ContentCardCommonProperties, b: ContentCardCommonProperties) => {
-  if (a.order && !b.order) {
-    return -1;
-  }
-  if (!a.order && b.order) {
-    return 1;
-  }
-  if ((!a.order && !b.order) || a.order === b.order) {
-    return b.createdAt - a.createdAt;
-  }
-  return (a.order || 0) - (b.order || 0);
+  if (a.order === b.order) return b.createdAt - a.createdAt;
+  if (a.order === undefined) return 1;
```

Comment:
order can be null?

Reply:
<img width="349" alt="Screenshot 2024-12-02 at 10 50 30" src="https://github.com/user-attachments/assets/76d8088a-3266-4fe2-a938-80b5d894b9dd">
It can be a number or undefined